src/Microsoft.DotNet.Cli.Utils/Muxer.cs

```diff
@@ -11,7 +11,7 @@ public class Muxer
 
         private string _muxerPath;
 
-        internal string SharedFxVersion
+        public string SharedFxVersion
```

Comment:
Please do not make this public.  This assembly is published as a NuGet package that others take a dependency on.  We are extremely strict about our public surface area in this assembly.